app/views/public/index.js.erb

```diff
@@ -4,4 +4,14 @@ $("<%=j render partial: "posts/list", locals: { posts:@posts } %>").appendTo($("
   $('#infinite-scrolling').remove();
 <% else %>
   $('#infinite-scrolling').html("<%=j link_to_next_page(@posts, 'See more', :remote => true) %>");
-<% end %>
\ No newline at end of file
+<% end %>
+
+$(window).scroll(function() {
+  var next_url = $("#infinite-scrolling a[rel='next']").attr('href');
+  if (( next_url ) && ((window.innerHeight + window.pageYOffset) >= document.body.offsetHeight))
+  {
+    $('#my-posts').show();
+    $.getScript(next_url);
+    return;
+  }
+});
```

Comment:
this seems to be repeated in many places. Can we just move this to one file and use it everywhere?